docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,179 @@
+# Allowing OpenShift Groups to be Synced With Sources on an LDAP Server and Making it Extensible for Other External Sources
```

Comment:
Do we care about extending this to other sources?  I don't see a reason to do that.